app/assets/tailwind/application.css

```diff
@@ -0,0 +1,75 @@
+@import 'tailwindcss';
+@import 'https://cdnjs.cloudflare.com/ajax/libs/tom-select/2.3.1/css/tom-select.min.css'
+  layer(utilities);
+
+@plugin '@tailwindcss/forms';
+@plugin '@tailwindcss/aspect-ratio';
+@plugin '@tailwindcss/typography';
+
+@theme {
+  --color-primary: rgb(18 18 18);
+  --color-secondary: #eeb14a;
+  --color-tertiary: rgb(165 68 29);
+  --color-input-label: rgb(238 177 74 / 73%);
+  --color-nav-backdrop: rgb(45 45 45);
+  --color-nav-link-bg: rgba(0, 0, 0, 0.4);
```

Comment:
The duplication of these from the tailwind.config.js is also not great

Reply:
I could remove the --color and name them how we had them if that works better.

Reply:
I'm not really concerned about the naming, I just see these same colors defined in 2 separate files. Are they both being used? If we want to update one, do both files need to be modified?

Reply:
So for this I removed the stylings from both files and nothing changed. The styles were still there. With both of them remoevd. So I am very confused here.